data_legacy/README.md

```diff
@@ -0,0 +1,101 @@
+# Drivy Data Challenge
```

Comment:
Not convinced that keeping safety copies of old files in a git repository makes a lot of sense.